include/dem/explicit_euler_integrator.h

```diff
@@ -54,15 +54,19 @@ class ExplicitEulerIntegrator : public Integrator<dim>
    * @param particle_handler The particle handler whose particle motion we wish
    * to integrate
    * @param body_force A constant volumetric body force applied to all particles
-   * @param force Force acting on particles
    * @param time_step The value of the time step used for the integration
+   * @param momentum Momentum of particles
+   * @param force Force acting on particles
+   * @param MOI A container of moment of inertia of particles
    */
   virtual void
```

Comment:
Should this function be virtual?

Reply:
yes, because it is rewritten by different integration methods.